api/src/services/competenciesService.ts

```diff
@@ -55,3 +55,20 @@ export const authorizeCompetencyUpdate = async (
 
   return !!competency;
 };
+
+export const getAllcompetenciesByCategory = async (categoryId: number) => {
```

Comment:
I think "competencies" in the function name should be capitalized.
Current: **getAllcompetenciesByCategory**
Change to: **getAllCompetenciesByCategory**